---
layout: cv
title: Nikolaus Sonnenschein's CV
---
# Nikolaus Sonnenschein

Associate Professor, Technical University of Denmark

<div id="webaddress">
<a href="nikosonnensche.in">nikosonnensche.in</a>
| <a href="mailto:niso@dtu.dk">niso@dtu.dk</a>
| <a href="https://github.com/phantomas1234"><i class="fa fa-github"></i></a>
<a href="https://twitter.com/phantomas1234"><i class="fa fa-twitter"></i></a>
<a href="https://dk.linkedin.com/in/nikosonnenschein"><i class="fa fa-linkedin"></i></a>
<a href="http://scholar.google.dk/citations?user=8RqKcm0AAAAJ&hl=en"><i class="fa fa-graduation-cap"></i></a>
<a href="mailto:niko.sonnenschein@gmail.com"><i class="fa fa-envelope"></i></a>
</div>

## Interests

`Biology` Metabolism, Meta

`Software` adfasdf

`Automation`
(Computational) Biology, Mathematical Modeling, Data Science, Software engineering, Lab and Data Automation

## Work

`Nov2019 - now`
__Associate Professor__, DTU Bioengineering, Technical University of Denmark

`Jul2016 - Sep2019`
__Senior Researcher__, The Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark

`Aug2013 - Jun2016`
__Research Scientist__, The Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark

`Mar2011 - Jul2013`
__Postdoctoral Researcher__, University of California, San Diego, USA

`Sep2007 - Jan2011`
__PhD Student__, Jacobs University Bremen, Germany

## Education

`Jan2011`
__Ph.D.in Bioinformatics__, Jacobs University Bremen, Germany

`Jul2007`
__Diplom in Biology__ (equivalent to M.Sc.), Technical University of Darmstadt, Germany

`May2004`
__Vordiplom in Biology__ (equivalent to B.Sc.), Technical University of Darmstadt, Germany


## Publications

A list is also available [online](http://scholar.google.co.uk/citations?user=LTOTl0YAAAAJ)

Lu H, Li F, Sánchez BJ, Zhu Z, Li G, Domenzain I, et al. A consensus S. cerevisiae metabolic model Yeast8 and its ecosystem for comprehensively probing cellular metabolism. Nat Commun. 2019;10: 3586. doi:10.1038/s41467-019-11581-3

Jessop-Fabre MM, Sonnenschein N. Improving Reproducibility in Synthetic Biology. Front Bioeng Biotechnol. 2019;7: 18. doi:10.3389/fbioe.2019.00018

Lennen RM, Jensen K, Mohammed ET, Malla S, Börner RA, Chekina K, et al. Adaptive laboratory evolution reveals general and specific chemical tolerance mechanisms and enhances biochemical production. bioRxiv. 2019. p. 634105. doi:10.1101/634105

Jensen PR, Matos MRA, Sonnenschein N, Meier S. Combined In-Cell NMR and Simulation Approach to Probe Redox-Dependent Pathway Control. Anal Chem. 2019;91: 5395–5402. doi:10.1021/acs.analchem.9b00660

Jessop-Fabre MM, Sonnenschein N. Improving Reproducibility in Synthetic Biology. Front Bioeng Biotechnol. 2019;7: 18. doi:10.3389/fbioe.2019.00018

Jensen K, Broeken V, Hansen ASL, Sonnenschein N, Herrgård MJ. OptCouple: Joint simulation of gene knockouts, insertions and medium modifications for prediction of growth-coupled strain designs. Metab Eng Commun. 2019;8: e00087. doi:10.1016/j.mec.2019.e00087

Lieven C, Beber ME, Olivier BG, Bergmann FT, Ataman M, Babaei P, et al. Memote: A community-driven effort towards a standardized genome-scale metabolic model test suite. bioRxiv. 2018. p. 350991. doi:10.1101/350991

Jessop-Fabre MM, Sonnenschein N. Improving Reproducibility in Synthetic Biology. Frontiers in Bioengineering and Biotechnology. 2019;7: 18. doi:10.3389/fbioe.2019.00018

Massaiu I, Pasotti L, Sonnenschein N, Rama E, Cavaletti M, Magni P, et al. Integration of enzymatic data in Bacillus subtilis genome-scale metabolic model improves phenotype predictions and enables in silico design of poly-γ-glutamic acid production strains. Microb Cell Fact. 2019;18: 3. doi:10.1186/s12934-018-1052-2

Lieven C, Petersen LAH, Jørgensen SB, Gernaey KV, Herrgard MJ, Sonnenschein N. A Genome-Scale Metabolic Model for Methylococcus capsulatus (Bath) Suggests Reduced Efficiency Electron Transfer to the Particulate Methane Monooxygenase. Front Microbiol. 2018;9: 2947. doi:10.3389/fmicb.2018.02947

Lieven C, Herrgård MJ, Sonnenschein N. Microbial Methylotrophic Metabolism: Recent Metabolic Modeling Efforts and Their Applications In Industrial Biotechnology. Biotechnol J. 2018;13: e1800011. doi:10.1002/biot.201800011

Lieven C, Beber ME, Olivier BG, Bergmann FT, Babaei P, Bartell JA, et al. Memote: A community-driven effort towards a standardized genome-scale metabolic model test suite. bioRxiv. 2018. p. 350991. doi:10.1101/350991

Cardoso JGR, Zeidan AA, Jensen K, Sonnenschein N, Neves AR, Herrgård MJ, et al. MARSI: metabolite analogues for rational strain improvement. Bioinformatics. 2018 [cited 8 Jun 2018]. doi:10.1093/bioinformatics/bty108

Cardoso JGR, Jensen K, Lieven C, Lærke Hansen AS, Galkina S, Beber M, et al. Cameo: A Python Library for Computer Aided Metabolic Engineering and Optimization of Cell Factories. ACS Synth Biol. 2018;7: 1163–1166. doi:10.1021/acssynbio.7b00423

Hansen ASL, Lennen RM, Sonnenschein N, Herrgård MJ. Systems biology solutions for biochemical production challenges. Curr Opin Biotechnol. 2017;45: 85–91. doi:10.1016/j.copbio.2016.11.018

Jensen K, G.R. Cardoso J, Sonnenschein N. Optlang: An algebraic modeling language for mathematical optimization. JOSS. 2017;2. doi:10.21105/joss.00139

Cardoso JGR, Andersen MR, Herrgard MJ, Sonnenschein N. Analysis of genetic variation and potential applications in genome-scale metabolic modeling. Front Bioeng Biotechnol. 2015;3: 13. doi:10.3389/fbioe.2015.00013

Sonnenschein N, Marr C, Hütt M-T. A topological characterization of medium-dependent essential metabolic reactions. Metabolites. 2012;2: 632–647. doi:10.3390/metabo2030632

King ZA, Dräger A, Ebrahim A, Sonnenschein N, Lewis NE, Palsson BØ. Escher: A Web Application for Building, Sharing, and Embedding Data-Rich Visualizations of Biological Pathways. PLoS Comput Biol. 2015;11: e1004321. doi:10.1371/journal.pcbi.1004321

Bordbar A, McCloskey D, Zielinski DC, Sonnenschein N, Jamshidi N, Palsson BO. Personalized Whole-Cell Kinetic Models of Metabolism for Discovery in Genomics and Pharmacodynamics. Cell Systems. 2015;1: 283–292. doi:10.1016/j.cels.2015.10.003

Bergdahl B, Sonnenschein N, Machado D, Herrgård M, Forster J. Genome-Scale Models BT - Fundamental Bioengineering. Fundamental Bioengineering. Wiley-VCH Verlag GmbH & Co. KGaA; 2015. pp. 143–182. doi:10.1002/9783527697441.ch06

Thiele I, Swainston N, Fleming RMT, Hoppe A, Sahoo S, Aurich MK, et al. A community-driven global reconstruction of human metabolism. Nat Biotechnol. 2013;31: 419–425. doi:10.1038/nbt.2488

Beber ME, Fretter C, Jain S, Sonnenschein N, Müller-Hannemann M, Hütt M-T. Artefacts in statistical analyses of network motifs: general framework and application to metabolic networks. J R Soc Interface. 2012;9: 3426–3435. doi:10.1098/rsif.2012.0490

Ebrahim A, Almaas E, Bauer E, Bordbar A, Burgard AP, Chang RL, et al. Do genome-scale models need exact solvers or clearer standards? Mol Syst Biol. 2015;11: 831–831. doi:10.15252/msb.20156157

Sonnenschein N, Hütt M-T, Stoyan H, Stoyan D. Ranges of control in the transcriptional regulation of Escherichia coli. BMC Syst Biol. 2009;3: 119. doi:10.1186/1752-0509-3-119

Sonnenschein N, Golib Dzib JF, Lesne A, Eilebrecht S, Boulkroun S, Zennaro M-C, et al. A Network Perspective on Metabolic Inconsistency. BMC Syst Biol. 2012;6: 41. doi:10.1186/1752-0509-6-41

Kildegaard KR, Hallström BM, Blicher TH, Sonnenschein N, Jensen NB, Sherstyk S, et al. Evolution reveals a glutathione-dependent mechanism of 3-hydroxypropionic acid tolerance. Metab Eng. 2014;26C: 57–66. doi:10.1016/j.ymben.2014.09.004

Sonnenschein N, Geertz M, Muskhelishvili G, Hütt M-T. Analog regulation of metabolic demand. BMC Syst Biol. 2011;5: 40. doi:10.1186/1752-0509-5-40

Machado D, Zhuang KH, Sonnenschein N, Herrgard MJ. Editorial: Current Challenges in Modeling Cellular Metabolism. Front Bioeng Biotechnol. 2015;3: 107. doi:10.3389/fcell.2015.00017

Sonnenschein R, Laaksonen A, Clementi E. The Treatment of Rotational Motion in Molecular Dynamics. J Comput Chem. 1986;7: 645–647.

Zhang J, Sonnenschein N, Pihl TPB, Pedersen KR, Jensen MK, Keasling JD. Engineering an NADPH/NADP+ redox biosensor in yeast. ACS Synth Biol. 2016; acssynbio.6b00135. doi:10.1021/acssynbio.6b00135


<!-- ### Footer

Last updated: October 2019 -->


