
[1] V\. Mol, M\. Bennett, B\. J\. Sánchez, B\. K\. Lisowska, M\. J\. Herrgård, A\. T\. Nielsen, D\. J\. Leak, and N\. Sonnenschein\.
Genome\-scale metabolic modeling of p\. thermoglucosidasius ncimb 11955 reveals metabolic bottlenecks in anaerobic metabolism\.
*Metabolic Engineering*, 65:123&ndash;134, 2021\.
[doi:https://doi\.org/10\.1016/j\.ymben\.2021\.03\.002](https://doi.org/https://doi.org/10.1016/j.ymben.2021.03.002)\.  

[2] S\. Li, C\. B\. Jendresen, J\. Landberg, L\. E\. Pedersen, **N. Sonnenschein**, S\. I\. Jensen, and A\. T\. Nielsen\.
Genome\-Wide CRISPRi\-Based identification of targets for decoupling growth from production\.
*ACS Synth\. Biol\.*, 9\(5\):1030–1040, May 2020\.  

[3] C\. Lieven, M\. E\. Beber, B\. G\. Olivier, F\. T\. Bergmann, M\. Ataman, P\. Babaei, J\. A\. Bartell, L\. M\. Blank, S\. Chauhan, K\. Correia, C\. Diener, A\. Dräger, B\. E\. Ebert, J\. N\. Edirisinghe, J\. P\. Faria, A\. M\. Feist, G\. Fengos, R\. M\. T\. Fleming, B\. García\-Jiménez, V\. Hatzimanikatis, W\. van Helvoirt, C\. S\. Henry, H\. Hermjakob, M\. J\. Herrgård, A\. Kaafarani, H\. U\. Kim, Z\. King, S\. Klamt, E\. Klipp, J\. J\. Koehorst, M\. König, M\. Lakshmanan, D\.\-Y\. Lee, S\. Y\. Lee, S\. Lee, N\. E\. Lewis, F\. Liu, H\. Ma, D\. Machado, R\. Mahadevan, P\. Maia, A\. Mardinoglu, G\. L\. Medlock, J\. M\. Monk, J\. Nielsen, L\. K\. Nielsen, J\. Nogales, I\. Nookaew, B\. O\. Palsson, J\. A\. Papin, K\. R\. Patil, M\. Poolman, N\. D\. Price, O\. Resendis\-Antonio, A\. Richelle, I\. Rocha, B\. J\. Sánchez, P\. J\. Schaap, R\. S\. Malik Sheriff, S\. Shoaie, **<u>N. Sonnenschein</u>**, B\. Teusink, P\. Vilaça, J\. O\. Vik, J\. A\. H\. Wodke, J\. C\. Xavier, Q\. Yuan, M\. Zakhartsev, and C\. Zhang\.
MEMOTE for standardized genome\-scale metabolic model testing\.
*Nat\. Biotechnol\.*, 38\(3\):272–276, March 2020\.  

[4] N\. R\. Wright, T\. Wulff, E\. A\. Palmqvist, T\. R\. Jørgensen, C\. T\. Workman, **N. Sonnenschein**, N\. P\. Rønnest, and M\. J\. Herrgård\.
Fluctuations in glucose availability prevent global proteome changes and physiological transition during prolonged chemostat cultivations of saccharomyces cerevisiae\.
*Biotechnol\. Bioeng\.*, 117\(7\):2074–2088, July 2020\.  

[5] K\. Jensen, V\. Broeken, A\. S\. Lærke Hansen, **N. Sonnenschein**, and M\. J\. Herrgård\.
OptCouple: joint simulation of gene knockouts, insertions and medium modifications for prediction of growth\-coupled strain designs\.
*Metabolic Engineering Communications*, pages e00087, March 2019\.
[doi:10\.1016/j\.mec\.2019\.e00087](https://doi.org/10.1016/j.mec.2019.e00087)\.  

[6] P\. R\. Jensen, M\. R\. A\. Matos, **N. Sonnenschein**, and S\. Meier\.
Combined In\-Cell NMR and simulation approach to probe Redox\-Dependent pathway control\.
*Anal\. Chem\.*, 91\(8\):5395–5402, April 2019\.
[doi:10\.1021/acs\.analchem\.9b00660](https://doi.org/10.1021/acs.analchem.9b00660)\.  

[7] M\. M\. Jessop\-Fabre and **<u>N. Sonnenschein</u>**\.
Improving reproducibility in synthetic biology\.
*Front Bioeng Biotechnol*, 7:18, February 2019\.
[doi:10\.1101/329714](https://doi.org/10.1101/329714)\.  

[8] H\. Lu, F\. Li, B\. J\. Sánchez, Z\. Zhu, G\. Li, I\. Domenzain, S\. Marcišauskas, P\. M\. Anton, D\. Lappa, C\. Lieven, M\. E\. Beber, **N. Sonnenschein**, E\. J\. Kerkhoven, and J\. Nielsen\.
A consensus s\. cerevisiae metabolic model yeast8 and its ecosystem for comprehensively probing cellular metabolism\.
*Nat\. Commun\.*, 10\(1\):3586, August 2019\.
[doi:10\.1038/s41467\-019\-11581\-3](https://doi.org/10.1038/s41467-019-11581-3)\.  

[9] I\. Massaiu, L\. Pasotti, **N. Sonnenschein**, E\. Rama, M\. Cavaletti, P\. Magni, C\. Calvio, and M\. J\. Herrgård\.
Integration of enzymatic data in bacillus subtilis genome\-scale metabolic model improves phenotype predictions and enables in silico design of poly\-γ\-glutamic acid production strains\.
*Microb\. Cell Fact\.*, 18\(1\):3, January 2019\.
[doi:10\.1186/s12934\-018\-1052\-2](https://doi.org/10.1186/s12934-018-1052-2)\.  

[10] J\. G\. R\. Cardoso, K\. Jensen, C\. Lieven, A\. S\. Lærke Hansen, S\. Galkina, M\. Beber, E\. Özdemir, M\. J\. Herrgård, H\. Redestig, and **<u>N. Sonnenschein</u>**\.
Cameo: a python library for computer aided metabolic engineering and optimization of cell factories\.
*ACS Synth\. Biol\.*, April 2018\.
[doi:10\.1021/acssynbio\.7b00423](https://doi.org/10.1021/acssynbio.7b00423)\.  

[11] J\. G\. R\. Cardoso, A\. A\. Zeidan, K\. Jensen, **N. Sonnenschein**, A\. R\. Neves, and M\. J\. Herrgård\.
MARSI: metabolite analogues for rational strain improvement\.
*Bioinformatics*, February 2018\.
[doi:10\.1093/bioinformatics/bty108](https://doi.org/10.1093/bioinformatics/bty108)\.  

[12] C\. Lieven, M\. J\. Herrgård, and **<u>N. Sonnenschein</u>**\.
Microbial methylotrophic metabolism: recent metabolic modeling efforts and their applications in industrial biotechnology\.
*Biotechnol\. J\.*, pages 1800011, June 2018\.
[doi:10\.1002/biot\.201800011](https://doi.org/10.1002/biot.201800011)\.  

[13] C\. Lieven, L\. A\. H\. Petersen, S\. B\. Jørgensen, K\. V\. Gernaey, M\. J\. Herrgard, and **N. Sonnenschein**\.
A Genome\-Scale metabolic model for methylococcus capsulatus \(bath\) suggests reduced efficiency electron transfer to the particulate methane monooxygenase\.
*Front\. Microbiol\.*, 9:2947, December 2018\.
[doi:10\.3389/fmicb\.2018\.02947](https://doi.org/10.3389/fmicb.2018.02947)\.  

[14] A\. S\. L\. Hansen, R\. M\. Lennen, **N. Sonnenschein**, and M\. J\. Herrgård\.
Systems biology solutions for biochemical production challenges\.
*Curr\. Opin\. Biotechnol\.*, 45:85–91, June 2017\.
[doi:10\.1016/j\.copbio\.2016\.11\.018](https://doi.org/10.1016/j.copbio.2016.11.018)\.  

[15] K\. Jensen, J\. G\.R\. Cardoso, and **N. Sonnenschein**\.
Optlang: an algebraic modeling language for mathematical optimization\.
*The Journal of Open Source Software*, jan 2017\.
[doi:10\.21105/joss\.00139](https://doi.org/10.21105/joss.00139)\.  

[16] J\. Zhang, **N. Sonnenschein**, T\. P\. B\. Pihl, K\. R\. Pedersen, M\. K\. Jensen, and J\. D\. Keasling\.
Engineering an NADPH/NADP\+ redox biosensor in yeast\.
*ACS Synth\. Biol\.*, pages acssynbio\.6b00135, 2016\.
[doi:10\.1021/acssynbio\.6b00135](https://doi.org/10.1021/acssynbio.6b00135)\.  

[17] B\. Bergdahl, **N. Sonnenschein**, D\. Machado, M\. Herrgård, and J\. Forster\.
Genome\-Scale Models \- Fundamental Bioengineering\.
In *Fundamental Bioengineering*, pages 143–182\.
Wiley\-VCH Verlag GmbH &amp; Co\. KGaA, 2015\.
[doi:10\.1002/9783527697441\.ch06](https://doi.org/10.1002/9783527697441.ch06)\.  

[18] A\. Bordbar, D\. McCloskey, D\. C\. Zielinski, **N. Sonnenschein**, N\. Jamshidi, and B\. Ø\. Palsson\.
Personalized Whole\-Cell Kinetic Models of Metabolism for Discovery in Genomics and Pharmacodynamics\.
*Cell Systems*, 1\(4\):283–292, oct 2015\.
[doi:10\.1016/j\.cels\.2015\.10\.003](https://doi.org/10.1016/j.cels.2015.10.003)\.  

[19] J\. G\. R\. Cardoso, M\. R\. Andersen, M\. J\. Herrgard, and **<u>N. Sonnenschein</u>**\.
Analysis of genetic variation and potential applications in genome\-scale metabolic modeling\.
*Frontiers in bioengineering and biotechnology*, 3:13, 2015\.
[doi:10\.3389/fbioe\.2015\.00013](https://doi.org/10.3389/fbioe.2015.00013)\.  

[20] A\. Ebrahim, E\. Almaas, E\. Bauer, A\. Bordbar, A\. P\. Burgard, R\. L\. Chang, A\. Dräger, I\. Famili, A\. M\. Feist, R\. M\. Fleming, S\. S\. Fong, V\. Hatzimanikatis, M\. J\. Herrgard, A\. Holder, M\. Hucka, D\. Hyduke, N\. Jamshidi, S\. Y\. Lee, N\. Le Novere, J\. A\. Lerman, N\. E\. Lewis, D\. Ma, R\. Mahadevan, C\. Maranas, H\. Nagarajan, A\. Navid, J\. Nielsen, L\. K\. Nielsen, J\. Nogales, A\. Noronha, C\. Pal, B\. Ø\. Palsson, J\. A\. Papin, K\. R\. Patil, N\. D\. Price, J\. L\. Reed, M\. Saunders, R\. S\. Senger, **N. Sonnenschein**, Y\. Sun, and I\. Thiele\.
Do genome\-scale models need exact solvers or clearer standards?
*Molecular Systems Biology*, 11\(10\):831–831, oct 2015\.
[doi:10\.15252/msb\.20156157](https://doi.org/10.15252/msb.20156157)\.  

[21] Z\. A\. King, A\. Dräger, A\. Ebrahim, **N. Sonnenschein**, N\. E\. Lewis, and B\. Ø\. Palsson\.
Escher: A Web Application for Building, Sharing, and Embedding Data\-Rich Visualizations of Biological Pathways\.
*PLoS computational biology*, 11\(8\):e1004321, 2015\.
[doi:10\.1371/journal\.pcbi\.1004321](https://doi.org/10.1371/journal.pcbi.1004321)\.  

[22] D\. Machado, K\. H\. Zhuang, **N. Sonnenschein**, and M\. J\. Herrgard\.
Editorial: Current Challenges in Modeling Cellular Metabolism\.
*Frontiers in bioengineering and biotechnology*, 3:107, 2015\.
[doi:10\.3389/fcell\.2015\.00017](https://doi.org/10.3389/fcell.2015.00017)\.  

[23] K\. R\. Kildegaard, B\. M\. Hallström, T\. H\. Blicher, **N. Sonnenschein**, N\. B\. Jensen, S\. Sherstyk, S\. J\. Harrison, J\. Maury, M\. J\. Herrgard, A\. S\. Juncker, J\. Forster, J\. Nielsen, and I\. Borodina\.
Evolution reveals a glutathione\-dependent mechanism of 3\-hydroxypropionic acid tolerance\.
*Metabolic engineering*, 26C:57–66, 2014\.
[doi:10\.1016/j\.ymben\.2014\.09\.004](https://doi.org/10.1016/j.ymben.2014.09.004)\.  

[24] I\. Thiele, N\. Swainston, R\. M\. T\. Fleming, A\. Hoppe, S\. Sahoo, M\. K\. Aurich, H\. Haraldsdottir, M\. L\. Mo, O\. Rolfsson, M\. D\. Stobbe, S\. G\. Thorleifsson, R\. Agren, C\. Bölling, S\. Bordel, A\. K\. Chavali, P\. Dobson, W\. B\. Dunn, L\. Endler, D\. Hala, M\. Hucka, D\. Hull, D\. Jameson, N\. Jamshidi, J\. J\. Jonsson, N\. Juty, S\. Keating, I\. Nookaew, N\. Le Novère, N\. Malys, A\. Mazein, J\. A\. Papin, N\. D\. Price, E\. Selkov, M\. I\. Sigurdsson, E\. Simeonidis, **N. Sonnenschein**, K\. Smallbone, A\. Sorokin, J\. H\. G\. M\. van Beek, D\. Weichart, I\. Goryanin, J\. Nielsen, H\. V\. Westerhoff, D\. B\. Kell, P\. Mendes, and B\. Ø\. Palsson\.
A community\-driven global reconstruction of human metabolism\.
*Nature biotechnology*, 31\(5\):419–425, 2013\.
[doi:10\.1038/nbt\.2488](https://doi.org/10.1038/nbt.2488)\.  

[25] **<u>N. Sonnenschein</u>**, C\. Marr, and M\.\-T\. Hütt\.
A topological characterization of medium\-dependent essential metabolic reactions\.
*Metabolites*, 2\(3\):632–647, 2012\.
[doi:10\.3390/metabo2030632](https://doi.org/10.3390/metabo2030632)\.  

[26] **<u>N. Sonnenschein</u>**, J\. F\. Golib Dzib, A\. Lesne, S\. Eilebrecht, S\. Boulkroun, M\.\-C\. Zennaro, A\. Benecke, and M\.\-T\. Hütt\.
A Network Perspective on Metabolic Inconsistency\.
*BMC systems biology*, 6\(1\):41, 2012\.
[doi:10\.1186/1752\-0509\-6\-41](https://doi.org/10.1186/1752-0509-6-41)\.  

[27] M\. E\. Beber, C\. Fretter, S\. Jain, **N. Sonnenschein**, M\. Müller\-Hannemann, and M\.\-T\. Hütt\.
Artefacts in statistical analyses of network motifs: general framework and application to metabolic networks\.
*Journal of the Royal Society, Interface / the Royal Society*, 9\(77\):3426–3435, 2012\.
[doi:10\.1098/rsif\.2012\.0490](https://doi.org/10.1098/rsif.2012.0490)\.  

[28] **<u>N. Sonnenschein</u>**, M\. Geertz, G\. Muskhelishvili, and M\.\-T\. Hütt\.
Analog regulation of metabolic demand\.
*BMC Systems Biology*, 5\(1\):40, 2011\.
[doi:10\.1186/1752\-0509\-5\-40](https://doi.org/10.1186/1752-0509-5-40)\.  

[29] **N. Sonnenschein**, M\.\-T\. Hütt, H\. Stoyan, and D\. Stoyan\.
Ranges of control in the transcriptional regulation of Escherichia coli\.
*BMC systems biology*, 3:119, 2009\.
[doi:10\.1186/1752\-0509\-3\-119](https://doi.org/10.1186/1752-0509-3-119)\.  

