---
layout: cv
title: Nikolaus Sonnenschein's CV
---
# Nikolaus Sonnenschein

Associate Professor at DTU Bioengineering, Computer Aided Biotechnology Group

<div id="webaddress">
<a href="nikosonnensche.in">nikosonnensche.in</a>
| <a href="mailto:niso@dtu.dk">niso@dtu.dk</a>
| <a href="https://github.com/phantomas1234"><i class="fa fa-github"></i></a>
<a href="https://twitter.com/phantomas1234"><i class="fa fa-twitter"></i></a>
<a href="https://dk.linkedin.com/in/nikosonnenschein"><i class="fa fa-linkedin"></i></a>
<a href="http://scholar.google.dk/citations?user=8RqKcm0AAAAJ&hl=en"><i class="fa fa-graduation-cap"></i></a>
<a href="mailto:niko.sonnenschein@gmail.com"><i class="fa fa-envelope"></i></a>
</div>

## Interests

`Biology` Metabolism, Cell factories, Metabolic modeling

`Automation` Cloud laboratoriers, Biofoundries, Reproducible research

`Software` Open Source Research Software, CAD for Biology, CI/CD, DevOps, SCRUM

## Professional Experience

`Nov2019 - now`
__Associate Professor__, DTU Bioengineering, Technical University of Denmark

`Jul2016 - Sep2019`
__Senior Researcher__, The Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark

`Aug2013 - Jun2016`
__Research Scientist__, The Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark

`Mar2011 - Jul2013`
__Postdoctoral Researcher__, University of California, San Diego, USA

`Sep2007 - Jan2011`
__PhD Student__, Jacobs University Bremen, Germany

`Oct2005 - Jul2006`
__Research Assistant__, Technical University of Darmstadt, Germany

`Feb2005 - Jun2006`
__Student Assistant__, Berlin-Brandenburgische Akademie der Wissenschaften, Germany

## Education

`Jan2011`
__PhD in Bioinformatics__, Jacobs University Bremen, Germany

`Jul2007`
__Diplom in Biology__ (equivalent to M.Sc.), Technical University of Darmstadt, Germany

`May2004`
__Vordiplom in Biology__ (equivalent to B.Sc.), Technical University of Darmstadt, Germany

## Oral Conference Presentations

`Nov 2019`
Invited speaker, European Summit of Industrial Biotechnology (ESIB 2019), Graz, Austria

`Oct2018`
Invited speaker, 5th Conference on Constraint-Based Reconstruction and Analysis (COBRA 2018), Seattle, WA, USA

`June2018`
Invited speaker, Metabolic Engineering X, Munich, Germany

`Apr2018`
Invited speaker, “Advancing Synthetic Biology” (mini-symposium), Beijing University of Chemical Technology, Beijing, China

`Dec2017`
Invited speaker, 4th Synthetic Biology & Gene Editing Congress, London, UK

`Aug2017`
Selected talk, 18th International Conference on Systems Biology (ICSB 2017), Blacksburg, VA, USA

`May2014`
Invited speaker, 3rd Conference on Constraint-Based Reconstruction and Analysis (COBRA 2014), Charlottesville, VA, USA

## Conference Organization

`Mar2019`
Organizer of Copenhagen Bioscience Cluster conference on “Software in the lifesciences: Development, Usability, Sustainability”

`2019`
Member of the International Programme Committee of the Foundation of Systems Biology in Engineering conference (FOSBE), Valencia, Spain

`2016`
Organization committee member and chairman, 11th Copenhagen Bioscience Conference, Favrholm, Denmark, “Data-driven Biotechnology – bench, bioreactor and bedside”. 

## Software contributions

Detailed contributions available also via <https://github.com/phantomas1234>.

`COBRApy`
Constraint-based modeling of genome-scale metabolic models in Python. We have been maintaining cobrapy for almost four years now. In that time we have been able to engage the community to get involved and make contributions. Cobrapy has furthermore become the basis for a number of more specialized packages (<https://opencobra.github.io/cobrapy/packages/>). 228 GitHub stars (<https://github.com/opencobra/cobrapy>).


`memote`
A test-suite and quality assessment tool for genome-scale metabolic models written in Python. The tests have been developed together with the COBRA community to achieve a community-wide consensus. A pre-print is available at bioRxiv (<https://doi.org/10.1101/350991>). The manuscript has just been accepted in Nature Biotechnology. 53 GitHub stars (<https://github.com/opencobra/memote>).


`cameo`
A Python package for computer aided metabolic engineering and optimization of cell factories. Cameo implements state-of-the-art methods for enumerating and prioritizing knockout, knock-in, overexpression, and down-regulation strategies and combinations thereof. Potential users can also give cameo a try at <https://try.cameo.bio>. Cameo has been published in ACS Synthetic Biology (<https://doi.org/10.1021/acssynbio.7b00423>). 74 GitHub stars (<https://github.com/biosustain/cameo>).


`optlang`
Optlang provides a common interface to a series of optimization softwares and relies on the popular symbolic mathematics package sympy for problem formulation (constraints, objectives, variables, etc.)\ and is now an essential part of COBRApy. Optlang has been adopted by researchers in a number of other fields. The tool has been published in the GitHub-based Journal of Open Source Software (<https://doi.org/10.21105/joss.00139>). GitHub stars 106 (<https://github.com/biosustain/optlang>).

## Certifications

`Aug2018-Aug2020`
Scrum Alliance, Certified Scrum Product Owner (CSPO), Credential ID 933770

## Selected publications

A list is also available here <https://goo.gl/DzH3My>. My name underlined indicates corresponding authorship.

### Articles

{% include_relative publications.md %}

<!-- ### Footer

Last updated: October 2019 -->


